source/declarations.tex

```diff
@@ -1558,7 +1558,7 @@
 \begin{codeblock}
 friend class T;
 \end{codeblock}
-is ill-formed. However, the similar declaration \tcode{friend T;} is allowed\iref{class.friend}.
+is ill-formed. However, the similar declaration \tcode{friend T;} is well-formed.\iref{class.friend}.
```

Comment:
@tkoeppe You introduced an extra period `.` here — the one after the `\iref` should stay but the new one before the `\iref` should be deleted again.